exporters/aerospike/aerospike.prometheus.json.tmpl

```diff
@@ -0,0 +1,43 @@
+{
+  "config_protocol_version": "1",
+  "action": "register_config",
+  "config_name": "cfg-aerospike",
+  "config": {
+    "variables": {},
+    "integrations": [
+         {
+            "name": "nri-prometheus",
+            {{- if .env.interval }}
+            "interval": "{{.env.interval}}",
+            {{ end }}
+            "config": {
+                "standalone": false,
+                {{- if .env.VERBOSE }}
+                "verbose": "{{.env.VERBOSE}}",
+                {{ end }}
+                "transformations":
+                [
+                    {{/* Adds transformation rules from the integration config file */}}
+                    {{ range $a := (fromJson .transformations)}}
+                        {{ toJson $a }},
+                    {{ end }}
+                    {{/* Add below any specific rules for this template. These will be hardcoded. */}}
+                    {"add_attributes":[{"prefix":"aerospike_","attributes":{"aerospike_cluster_name": "{{.config.aerospike_cluster_name}}" }}]}
+                ],
+                "integration_metadata":{
```

Comment:
we are adding the scrape timeout support and it would make sense to add it here as well. You can check as reference this [PR](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/164)

Reply:
done 👍